src/utils/renameImport.js

```diff
@@ -36,7 +36,10 @@ export function updateImport (j, root, { oldValue, newValue }) {
   root
     .find(j.ImportDeclaration)
     .filter(p => {
-      return p.parent.node.type === 'Program' && p.get('specifiers', 0, 'type').value !== 'ImportNamespaceSpecifier';
+      return  p.parent.node.type === 'Program' &&
+              p.get('specifiers', 0, 'type').value !== 'ImportNamespaceSpecifier' &&
+              p.value.source.value === 'can' ||
+              /\/\/unpkg.com\/can@5?(\.[0-9]+)?(\.[0-9]+)?\/everything.mjs/.test(p.value.source.value);
```

Comment:
A couple more pointers on this regex:
- `5?` doesn't need the `?` since the `5` should always be there.
- `Component` can be imported from `core.mjs` and `ecosystem.mjs` as well, so the regex should have `(core|ecosystem|everything)\.mjs`, or if you want to be even more permissive and concise, `[a-z]+\.mjs`

Reply:
Done, thanks!